android/2023-emmsale/app/src/main/java/com/emmsale/presentation/common/extension/EditTextExt.kt

```diff
@@ -0,0 +1,18 @@
+package com.emmsale.presentation.common.extension
+
+import android.content.Context.INPUT_METHOD_SERVICE
+import android.view.inputmethod.InputMethodManager
+import android.widget.EditText
+
+private const val DELAY_SHOW_SOFT_INPUT: Long = 200
+
+fun EditText.showKeyboard() {
+    val imm = context.getSystemService(INPUT_METHOD_SERVICE) as InputMethodManager
+    postDelayed(
```

Comment:
해당 코드를 사용해보시는 것을 추천드립니다.

```suggestion
fun Activity.showKeyboard() {
    val imm = getSystemService(AppCompatActivity.INPUT_METHOD_SERVICE) as InputMethodManager
    currentFocus?.postDelayed({
        imm.showSoftInput(currentFocus, InputMethodManager.SHOW_IMPLICIT)
    }, 100)
}

Reply:
와우! `currentFocus`라는 게 있군요. 그리고 저희는 사실 포커스 된 뷰에 텍스트를 편집하기 위해 소프트 키보드를 보여주는 것이므로 제안하신 코드가 훨씬 좋은 것 같네요! 👍